### ISSUE TITLE
Plasma crash with Qt 5.8 

### ISSUE THREAD

**Issue by ShalokShalom** [kotelnik/plasma-applet-redshift-control#31]
There are recent changes in Qt 5.8 due to QtFont and they seem to let the Redshift applet crash the Plasmashell, when you move the panel from the side to the top/bottom. 

Resizing the panel works fine. 

http://code.qt.io/cgit/qt/qtbase.git/tree/dist/changes-5.8.0
https://gist.github.com/ShalokShalom/7448ebf25e0309f8a353620b1ad92d1e

(By the way, v1.0.16 without the new translation is currently the freshest version in kde store:
https://store.kde.org/p/998916)

**Comment by kotelnik** [comment]
Hi! Thanks for this report. I can reproduce it but for now I don't know ho to fix it. It seems to me like a problem in Qt. I'll investigate further and find/create a bug report.

Thanks for reminding me the kde store, I'll update the widget as soon as possible.

**Comment by ShalokShalom** [comment]
Yes, its Qt related, afaik. They change something in 5.8, as linked.

**Comment by cfeck** [comment]
Is this https://bugreports.qt.io/browse/QTBUG-60697 ? If yes, Qt developers would require more information to reproduce.

**Comment by jtflynnz** [comment]
Seems to still be present in 5.10.1-1 on Arch; I have worked around this by triggering redshift from the terminal, or by using the menu-entry created by redshift-qt, but these bypass having a GUI completely. Has anyone found a workaround for this applet?
